build.gradle

```diff
@@ -1,38 +1,15 @@
-/*
- * Gradle build file for the RM Superpeer. Generated initially with
- * gradle 4.1 and the init command.
- * 
- * Uses a git plugin to checkout the Ethereum Parity Client vagrant
- * setup. Uses a plugin to execute the ethereum parity vagrant using
- * vagrant up. Parity client will be setup to listen on 
- * http://localhost:8545 so that the Superpeer can relay ethereum
- * remote signed transaction into it to be executed on the ethereum
- * blockchain. Should be configured to use either the Kovan testnet
- * or the public ethereum network.
- *
- * Requirements: git, vagrant 2.0.1 or higher, virtualbox
- * Note: if parity comes up correctly, you should be able to test it correctly
- * outside of the VM as follows: curl --data '{"method":"eth_accounts","params":[],"id":1,"jsonrpc":"2.0"}' -H "Content-Type: application/json" -X POST localhost:8545'
- * and you shouldn't get an error about connections.
- */
-//import com.bmuschko.gradle.vagrant.tasks.Vagrant
-//import com.bmuschko.gradle.vagrant.tasks.VagrantUp
-//import com.bmuschko.gradle.vagrant.tasks.VagrantSsh
-//import com.bmuschko.gradle.vagrant.tasks.VagrantDestroy
-
-// Apply the java plugin to add support for Java
-apply plugin: 'java'
-
-// Apply the application plugin to add support for building an application
-apply plugin: 'application'
-
-applicationDefaultJvmArgs = ["-noverify"]
 buildscript {
     ext {
-        //virtualBoxDir = file('build/parity')
-        artifactory_app_username = "username"
-        artifactory_app_password = "password"
-        artifactory_app_key = "app_key"
+        // define these variables in your personal ~/.gradle/gradle.properties
+        // for example:
+        /*
+            rightmesh_artifactory_username=<username>
+            rightmesh_artifactory_password=<hash of your developer portal password>
+            rightmesh_superpeer_key=<some key that you have access to>
+        */
+        artifactory_app_username = "${rightmesh_artifactory_username}"
```

Comment:
Does this work with `artifactory_app_{username, password, key}` also?

Reply:
Yes that should be fine. I think credentials can still sometimes be a pain to manage when working with artifactory-internal, so there's room for improvement on that front here. Just wanted to get this up to clean a lot of unused stuff out of the buildscript.